docs/data/charts/radar/radar.md

```diff
@@ -49,6 +55,7 @@ The radar chart displays a grid behind the series that can be configured with:
 - `startAngle` The rotation angle of the entire chart in degrees.
 - `divisions` The number of divisions of the grid.
 - `shape` The grid shape that can be `circular` or `sharp`.
+- `getStripeColor` The call back that defines stripe colors. Set it to `null` to remove stripes.
```

Comment:
`getStripeColor` is probably not a very good prop name. It works internally, but feels odd as a prop. 

Can't we use just `stripeColor`?